docs/guide/content_rendering.md

```diff
@@ -1384,30 +1290,30 @@ integer **number\_of\_bytes**|ez\_file\_size( integer **number\_of\_decimal** )
 | Argument name | Type                                                   | Description                               |
 |---------------|--------------------------------------------------------|-------------------------------------------|
 | `field`       | `eZ\Publish\API\Repository\Values\Content\Field`       | The image Field                           |
-| `versionInfo` | `eZ\Publish\API\Repository\Values\Content\VersionInfo` | The VersionInfo that the field belongs to |
-| `variantName` | `string`                                               | Name of the image alias to be used        |
+| `versionInfo` | `eZ\Publish\API\Repository\Values\Content\VersionInfo` | The VersionInfo that the Field belongs to |
+| `variantName` | `string`                                               | Name of the image variation to be used        |
```

Comment:
I suppose it's validated with the Repository Team.

Reply:
Existing doc used both "alias" and "variation" interchangeably and I tried to unify the terminology in https://github.com/ezsystems/developer-documentation/pull/56, so this is a follow-up
But there are two reasons to stick with variation. First, it's actually in the name of the key used in the config: https://github.com/ezsystems/DemoBundle/blob/master/Resources/config/image_variations.yml#L3. Second: "alias" is already user in other domain terms ("URLAlias" for example), so this avoids misunderstandings.

Reply:
I prefer also variation, my question was about the code and references to it. So thank you for your investigation!